pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -1 +1,37 @@
-<div></div>
\ No newline at end of file
+<section>
+  <h2 class="text-xl font-bold">Secure Envelopes</h2>
+  {{ if .SecureEnvelopes }}
+  {{ range .SecureEnvelopes }}
+  <div class="join join-vertical w-full">
+    <div class="collapse collapse-plus join-item border border-base-300">
+      <input type="radio" name="my-accordion-4" checked="checked" />
+      <div class="collapse-title text-xl font-medium">
+        <div class="flex items-center gap-x-1">
+          {{ if eq .Direction "in" }}
```

Comment:
These are the same icons displayed on the transaction list page. I'd like to use an alternative, but didn't quite find what I was looking for on FA. I can look at other sites in the future.